office/zathura/Portfile

```diff
@@ -77,6 +82,12 @@ subport ${name}-docs {
     unset notes
 }
 
+variant synctex description "Enable synctex support" {
+    depends_lib-append  port:texlive-bin
+
+    configure.args-append   -Dsynctex=enabled
```

Comment:
You want to remove the other argument. This is how to do that:

`configure.args-replace   -Dsynctex=disabled -Dsynctex=enabled
`

Reply:
Oh, that's exactly what I was looking for, thanks!